src/couch_views/test/couch_views_indexer_test.erl

```diff
@@ -388,8 +384,8 @@ multiple_design_docs(Db) ->
 
     % This is how we check that no index updates took place
     meck:new(couch_views_fdb, [passthrough]),
-    meck:expect(couch_views_fdb, write_doc, fun(TxDb, Sig, ViewIds, Doc) ->
-        meck:passthrough([TxDb, Sig, ViewIds, Doc])
+    meck:expect(couch_views_fdb, update_views, fun(TxDb, Mrst, Docs) ->
```

Comment:
This is confusing.  We create an `expect` for an `update_views`, but use `meck:num_calls/3` on `write_doc`.

Reply:
Good catch. write_doc was renamed and I forgot to update the test. Will update with a fix.

Reply:
Fixed.